### ISSUE TITLE
Do we need to update the versions in YCSB's README?

### ISSUE THREAD

**Issue by srmadscience** [brianfrankcooper/YCSB#1362]
Currently  https://github.com/brianfrankcooper/YCSB/blob/master/README.md says:

curl -O --location https://github.com/brianfrankcooper/YCSB/releases/download/0.15.0/ycsb-0.15.0.tar.gz
tar xfvz ycsb-0.15.0.tar.gz
cd ycsb-0.15.0

Should it say:

curl -O --location https://github.com/brianfrankcooper/YCSB/releases/download/0.17.0/ycsb-0.17.0.tar.gz
tar xfvz ycsb-0.17.0.tar.gz
cd ycsb-0.17.0

If you want I can do this, but (1) I don't know if you have a good reason for mentioning 15, and (2) it might not justify a full dev cycle...

David Rolfe

**Comment by busbey** [comment]
Yep! That's part of the cleanup that needs to be done after a release goes out.

Now that you mention it, I'm not sure why we doing just have those instructions guide folks to the "latest" release pointer. It should always get to the most recent production release.
